keyform/static/keyform/css/request.css

```diff
@@ -6,7 +6,7 @@
     margin-top: .5em;
 }
 
-.content {
+.content, .hide-for-print {
```

Comment:
The 'hide-for-print' class shouldn't have these css attributes associated with it. It would be better to instead add the 'content' class to the elements which you would like them to apply.